package.json

```diff
@@ -18,6 +18,8 @@
     "big.js": "^5.2.2",
     "bip39": "^3.0.2",
     "crypto-js": "^4.0.0",
+    "long": "^5.2.0",
```

Comment:
The generated files depend on `long` and `protobufjs`. 

`long` is used for 64 bit numbers
`protobufjs` is used for serialization/deserialization